draft-ietf-ppm-dap-taskprov.md

```diff
@@ -429,12 +429,12 @@ Before a task can be executed, it is necessary to first provision the Clients,
 Aggregators, and Collector with the task's configuration. The core DAP
 specification does not define a mechanism for provisioning tasks. This section
 describes a mechanism whose key feature is that task configuration is
-performed completely in-band, via HTTP request headers.
+performed completely in-band in an HTTP Structured Header {{!RFC9651}}.
```

Comment:
You can just say "header field" for these.  You only really need the Formal Name for when you define it.

Reply:
Done.